### ISSUE TITLE
Release trained SNN models (MP-Init, TrSG) on Hugging Face

### ISSUE THREAD

**Issue by NielsRogge** [kookhh0827/SNN-MP-Init-TRSG#1]
Hi @kookhh0827 🤗

I'm Niels and work as part of the open-source team at Hugging Face. I discovered your work on Arxiv and your paper "Stabilizing Direct Training of Spiking Neural Networks: Membrane Potential Initialization and Threshold-robust Surrogate Gradient" got featured on our daily papers page: https://huggingface.co/papers/2511.08708.

The paper page lets people discuss your research and find related artifacts, such as models and datasets. You can also claim the paper as yours, which will link it to your public profile on HF, and add GitHub and project page URLs.

I noticed that your abstract mentions code availability at https://github.com/kookhh0827/SNN-MP-Init-TRSG, but the repository currently contains a minimal README. I was wondering if you plan to release the pre-trained SNN models incorporating your MP-Init and TrSG innovations?

It would be fantastic to host these models on https://huggingface.co/models! Hosting on Hugging Face provides greater visibility and discoverability for your work. We can add relevant tags to the model cards (e.g., `image-classification`) to make them easier to find and link them directly to your paper page.

If you're interested, here's a guide on uploading models: https://huggingface.co/docs/hub/models-uploading. For custom PyTorch models, the [PyTorchModelHubMixin](https://huggingface.co/docs/huggingface_hub/package_reference/mixins#huggingface_hub.PyTorchModelHubMixin) class simplifies the process by adding `from_pretrained` and `push_to_hub` functionalities. Alternatively, users can download models directly using [hf_hub_download](https://huggingface.co/docs/huggingface_hub/en/guides/download#download-a-single-file).

We can also help link the models to your paper page (read [here](https://huggingface.co/docs/hub/en/model-cards#linking-a-paper)) once uploaded.

You could also consider building a demo for your model on [Spaces](https://huggingface.co/spaces), and we can potentially offer a ZeroGPU [grant](https://huggingface.co/docs/hub/en/spaces-gpus#community-gpu-grants) for free A100 GPUs.

Please let me know if you're interested or need any guidance!

Kind regards,

Niels
ML Engineer @ HF 🤗

**Comment by kookhh0827** [comment]
Thank you for your interest in our work. We plan to reorganize the code and release the trained models for reproduction purposes.
